crowns.lic

```diff
@@ -0,0 +1,85 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#dice-game
+=end
+
+arg_definitions = [
+  [
+    { name: 'bet', regex: /\d+/, optional: true, description: 'Optional override for betting amount.' }
+  ]
+]
+
+custom_require.call(%w[events common-travel common])
+
+settings = get_settings
+junk_list = settings.darkbox_junk.map { |x| /#{x}/i }
+dice_bet_amount = settings.dice_bet_amount || args.bet
```

Comment:
`dice_bet_amount` needs to be in `base.yaml`

Reply:
Added to base.